src/test/java/emissary/core/IBaseDataObjectXmlHelperTest.java

```diff
@@ -81,7 +82,7 @@ void testParentIbdoAllFieldsChanged() throws Exception {
         expectedIbdo.setTransactionId("TransactionId");
         expectedIbdo.setWorkBundleId("WorkBundleId");
         expectedIbdo.putParameter("Parameter1Key", "Parameter1Value");
-        expectedIbdo.putParameter("Parameter2Key", "Parameter2Value");
+        expectedIbdo.putParameter("Parameter2Key", new ArrayList<Object>(Arrays.asList(new Object[] {"Parameter2Value1", "Parameter2Value2"})));
```

Comment:
Can remove redundant List wrapper 
```suggestion
        expectedIbdo.putParameter("Parameter2Key", Arrays.asList(new Object[] {"Parameter2Value1", "Parameter2Value2"}));
```